submissions/dashakim/memory-game/script.js

```diff
@@ -1,90 +1,96 @@
-const section = document.querySelector("section");
+const section = document.querySelector("section.memory-game");
 
-const imagesNames = function(currentImg) {
+const getImagesNamesForAltAttribute = function(currentImg) {
```

Comment:
At this iteration, it's good enough. No need to rewrite it.

I just wanted to pay your attention: `ForAltAttribute` is the redundant part of the function name because describes the narrow applying (single purpose). Also, your function works with a single image, not an array of images. So `getImageName` would be more clear.
By code writing, variables (and functions, etc.) naming talks about your level a much. Since you just started - it will be very useful to take some time and think more about the successful variable/function name. Take it for a good habit - it will pay off.